R/geojson_list.R

```diff
@@ -11,6 +11,9 @@
 #' GeometryCollection.
 #' @param group (character) A grouping variable to perform grouping for polygons - doesn't apply
 #' for points
+#' @param precision desired number of decimal places for the coordinates in the
+#'   geojson file. Using fewer decimal places can decrease file sizes (at the
+#'   cost of precision).
```

Comment:
I think we need to add here that this works up to a point, that point being the value of `getOption("digits")`. If you want more digits than `precision` allows, then fiddle with `options(digits = <some number>)`